chrome/skin/style.css

```diff
@@ -154,6 +154,13 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
   background-image: linear-gradient(rgba(255,255,255,0.3), transparent);
   color: HighlightText;
 }
+#universal-search-recommendation.highlight [data-result-type="wikipedia"] #universal-search-recommendation-icon {
```

Comment:
It looks like this data- attribute is only used for styling. Let's use a classname instead. If we need to get the result type, we can get it on the JS side (`recommendation.resultType`) without paying the high price of accessing the DOM.